@app/config/src/index.ts

```diff
@@ -3,11 +3,170 @@ const packageJson = require("../../../package.json");
+    );
+  }
+  return v as any;
+};
+
+const minLength = (
+  v: { value: string; name: string },
+  minLength: number
+): { value: string; name: string } => {
+  const { value, name } = v;
+  if (value.length < minLength) {
+    throw new Error(
+      `process.env.${name} should have minimum length ${minLength}.`
+    );
+  }
+  return v;
+};
+
+const parseInteger = (v: {
+  value?: string;
+  name: string;
+}): { value: number; name: string } => ({
+  value: parseInt(v.value || "", 10),
+  name: v.name,
+});
+
+const defaultNumber = (
+  v: { value?: number; name: string },
+  defaultValue: number
+): { value: number; name: string } => {
+  const { value, name } = v;
+  return {
+    value: value === undefined || isNaN(value) ? defaultValue : value,
+    name,
+  };
+};
+
+export const SECRET_COOKIE: string = minLength(
+  required({ value: process.env.SECRET_COOKIE, name: "SECRET_COOKIE" }),
+  40
+).value;
+
+export const SECRET_JWT_ACCESS: string = minLength(
+  required({
+    value: process.env.SECRET_JWT_ACCESS,
+    name: "SECRET_JWT_ACCESS",
+  }),
+  40
+).value;
+
+export const SECRET_JWT_REFRESH: string = minLength(
+  required({
+    value: process.env.SECRET_JWT_REFRESH,
+    name: "SECRET_JWT_REFRESH",
+  }),
+  40
+).value;
+
+export const REDIS_URL: string | undefined = process.env.REDIS_URL;
+
+export const DATABASE_URL: string = required({
+  value: process.env.DATABASE_URL,
+  name: "DATABASE_URL",
+}).value;
+
+export const AUTH_DATABASE_URL: string = required({
+  value: process.env.AUTH_DATABASE_URL,
+  name: "AUTH_DATABASE_URL",
+}).value;
+
+export const NODE_ENV: string = required({
+  value: process.env.NODE_ENV,
+  name: "NODE_ENV",
+}).value;
+
+export const isDev: boolean = NODE_ENV === "development";
+
+export const isTest: boolean = NODE_ENV === "test";
+
+export const isDevOrTest: boolean = isDev || isTest;
+
+export const ROOT_URL: string = required({
+  value: process.env.ROOT_URL,
+  name: "ROOT_URL",
+}).value;
+
+export const TRUST_PROXY: string | undefined = process.env.TRUST_PROXY;
+
+export const ENABLE_GRAPHIQL: boolean = !!process.env.ENABLE_GRAPHIQL;
+
+export const GRAPHILE_LICENSE: string | undefined =
+  process.env.GRAPHILE_LICENSE;
+// Pro plugin options (requires process.env.GRAPHILE_LICENSE)
+export const GRAPHQL_PAGINATION_CAP: number = defaultNumber(
+  parseInteger({
+    value: process.env.GRAPHQL_PAGINATION_CAP,
+    name: "GRAPHQL_PAGINATION_CAP",
+  }),
+  50
+).value;
+export const GRAPHQL_DEPTH_LIMIT: number = defaultNumber(
+  parseInteger({
+    value: process.env.GRAPHQL_DEPTH_LIMIT,
+    name: "GRAPHQL_DEPTH_LIMIT",
+  }),
+  12
+).value;
+export const GRAPHQL_COST_LIMIT: number = defaultNumber(
+  parseInteger({
+    value: process.env.GRAPHQL_COST_LIMIT,
+    name: "GRAPHQL_COST_LIMIT",
+  }),
+  30000
+).value;
+export const HIDE_QUERY_COST: boolean =
```

Comment:
I don't really understand what this one was doing in the original code, but I think this functions the same.